Bluetooth: Mesh: Make Sensor Observer fit nrf52832 (#11451)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Bluetooth mesh samples
----------------------

* :ref:`bluetooth_mesh_sensor_client` sample:

>   * Fixed an issue where the sample would not fit in RAM on the nrf52dk_nrf52832 board.

Reviewer: ```suggestion
  * Fixed an issue with the sample not fitting into RAM size on the nrf52dk_nrf52832 board.
```
